demo/site/src/blocks/CookieSafeYouTubeVideoBlock.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+import { CookieSafe, PropsWithData, useCookieApi, YouTubeVideoBlock } from "@comet/cms-site";
+import { YouTubeVideoBlockData } from "@src/blocks.generated";
+import { CookiePlaceholder } from "@src/components/common/CookiePlaceholders";
+import { cookieIds } from "@src/util/cookieIds";
+import styled from "styled-components";
+
+const aspectRatio = "16x9";
+
+export const CookieSafeYouTubeVideoBlock = (props: PropsWithData<YouTubeVideoBlockData>) => {
```

Comment:
Don't you think it is possible (and desired) to implement a CookieSafe YouTube block in the library?

EDIT: "Yes, that's a follow-up task: AG-378" :+1: